lib/portlayer/attach/client.go

```diff
@@ -111,6 +113,22 @@ func (t *attachSSH) Signal(signal ssh.Signal) error {
 	return nil
 }
 
+func (t *attachSSH) CloseStdin() error {
+	defer trace.End(trace.Begin(""))
+
+	msg := msgs.CloseStdinMsg{Noop: true}
+	ok, err := t.channel.SendRequest(msgs.CloseStdinReq, false, msg.Marshal())
+	if err == nil && !ok {
+		return fmt.Errorf("unknown error")
```

Comment:
let's change this to "unknown error closing stdin" - otherwise if the string appears many levels of call stack from here and is logged it's going to be hard to determine where/why.